src/components/organisms/modal/Modal.jsx

```diff
@@ -1,16 +1,143 @@
+    lastName: "",
+    email: "",
+    attendance: "",
+    dietaryType: "Regular menu",
+    companions: "",
+    other: "",
+  });
+  //HANDLE FUNCTIONS
+  const handleChange = e => {
+    const { name, value } = e.target;
+    setFormData({
+      ...formData,
+      [name]: value,
+    });
+  };
+  const handleSubmit = e => {
+    e.preventDefault();
+    console.log(formData);
+  };
+  //MAIN RENDER
   return (
     <ModalWrapper>
       <ModalContent>
         <CloseButton onClick={onClose}>
           <span className="material-symbols-outlined">close_small</span>
         </CloseButton>
-        <h2>CONFIRM ATTENDANCE</h2>
-        {/* TODO: FORM */}
+        <ModalHeader>
+          <h1>{translate("modal.text.1")}</h1>
+          <p>{translate("modal.text.2")}</p>
+        </ModalHeader>
+        <Form onSubmit={handleSubmit}>
+          <label>
+            {translate("modal.text.3")}
+            <input
+              type="text"
+              name="firstName"
+              value={formData.firstName}
+              onChange={handleChange}
+              required
+            />
+          </label>
+          <br />
+          <label>
+            {translate("modal.text.4")}
+            <input
+              type="text"
+              name="lastName"
+              value={formData.lastName}
+              onChange={handleChange}
+              required
+            />
+          </label>
+          <br />
+          <label>
+            {translate("modal.text.5")}
+            <input
+              type="text"
+              name="email"
+              value={formData.email}
+              onChange={handleChange}
+              required
+            />
+          </label>
+          <br />
+
+          <RadioGroup>
+            <label>{translate("modal.text.6")}</label>
+            <RadioLabel>
+              <input
+                type="radio"
+                name="attendance"
+                value="yes"
+                checked={formData.attendance === "yes"}
+                onChange={handleChange}
+                required
+              />
+              <span></span>
+              {translate("modal.text.7")}
+            </RadioLabel>
+            <RadioLabel>
+              <input
+                type="radio"
+                name="attendance"
+                value="no"
+                checked={formData.attendance === "no"}
+                onChange={handleChange}
+                required
+              />
+              <span></span>
+              {translate("modal.text.8")}
+            </RadioLabel>
+          </RadioGroup>
+
+          <br />
+          <label>
+            {translate("modal.text.9")}
+            <select
+              name="dietaryType"
+              value={formData.dietaryType}
+              onChange={handleChange}
+            >
+              <option value="Regular menu">{translate("modal.text.10")}</option>
+              <option value="Gluten-free">{translate("modal.text.11")}</option>
```

Comment:
please make sure the values are like this :

`GLUTEN_FREE`

Reply:
Should I write the 3 values in capital letters?  (GLUTEN_FREE, REGULAR MENU, VEGAN)

Reply:
I fixed the errors, is ready to review it

Reply:
you did it wrong. you changed the `translations`